modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1392,13 +1392,6 @@ private void _processAttachment(
 			return;
 		}
 
-		if ((fileEntry.getId() != null) &&
```

Comment:
@devpedrotavares , based on the exception message, `getId` and `getFileBase64` should both be null to throw the error, right?

Reply:
I ended up removing this exception in this [commit](https://github.com/liferay-objects/liferay-portal/pull/3617/commits/7e7bfb22f4abfadaef42d0e3c2d893fbaa8f8c2b). Sorry for the confusion. 

But the "or" in the exception message was exclusive, so it was being thrown when the id and the base64 were present at the same time.